datafusion/core/tests/sql/information_schema.rs

```diff
@@ -707,8 +707,11 @@ async fn show_all() {
         "| datafusion.execution.coalesce_batches                     | true    |",
         "| datafusion.execution.coalesce_target_batch_size           | 4096    |",
         "| datafusion.execution.parquet.enable_page_index            | false   |",
+        "| datafusion.execution.parquet.metadata_size_hint           | NULL    |",
```

Comment:
🎉  one can now see much more explicitly both 1) what the parquet options are and 2) what their default values are